frontend/src/container/ReduxTlmDispatcher.jsx

```diff
@@ -91,10 +92,16 @@ export class ReduxTlmDispatcher extends React.Component {
     if (response.status !== 200) return
 
     this.props.dispatch(removeWorkspaceReadStatus(response.json, data.workspace.workspace_id))
+    if (data.author.user_id === this.props.user.userId) {
+      this.props.dispatch(addWorkspaceReadStatus(data.content, data.workspace.workspace_id))
+    }
```

Comment:
this part doesn't make a lot of sense
it literally says:
1) remove read status
2) if you are the author
3) add read status

Also, removeWorkspaceReadStatus() use the data from `response.json` and addWorkspaceReadStatus() from the TLM `data.content`

you are in a comment creation, the content is the comment, not the parent's content

And finally, from what I said yesterday, you are not supposed to add intelligence in here, if you really have to, tell me why please

(by "intelligence", I'm only talking about the if)

Reply:
Yes sorry, I updated the pull request according to preview comment too quickly

I put the 'intelligence' here because we must have the current user ID in order to know if the author is the current user connected to read the new revision. I put there because 2 components use `contentReadStatusList` but we could put additional TLM handlers in these 2 components

Reply:
ok, the reason to put the intelligence here because you need the connected user seems fine to me